android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -257,24 +276,25 @@ public void onError(int i) {
     }
 
     private boolean createLocalVideo(boolean enableVideo) {
-        if (!enableVideo) {
```

Comment:
@xaviraol I think this is the reason why running audio call for the first time on your branch results in app crashing, no?

Reply:
I see, this was happening at some moment, but not anymore I think. I'm not sure because I've been testing EMDR with video call.. Let me double-check today, or have you already?

Reply:
Last time I check yours in previous week, it's still happening.. but, won't hurt to get your confirmation. Just be sure to uninstall the previous version first.